### DISCUSSION TITLE
下载的缓存可以缓存后直接合并吗

### DISCUSSION THREAD

**Discussion by kettly1260** [ywmoyue/biliuwp-lite#484] [feature]
### 问题描述

缓存下载完都是M4SG格式的，而且还是分开的，能不能下完之后自动合并或者改一下扩展名成mp4/aac啊

### 描述解决方案

能不能下完之后自动合并或者改一下扩展名成mp4/aac啊

### 代码贡献

否

**Comment by GD-Slime** [comment]
目前好像是没有自动合并选项的。不过可以在下载完成后，右击下载完成的视频，并点击`导出视频`来导出常规格式。
![image](https://github.com/ywmoyue/biliuwp-lite/assets/82302542/1b3940c3-27eb-40ca-81f1-d7c524c0a70f)

**Reply by kettly1260** [reply]
看到了这个功能，但是对多P文件的导出很不方便
